cvmfs-mon/cvmfs-mon.py

```diff
@@ -44,7 +44,6 @@ def check(monit):
     try:
       api_url = monit["api_url"] + "/" + repo
       s = get(api_url).json()
-      ok = s["status"] == "ok"
```

Comment:
On a second thought. What happens if s does not have "status"? Is that actually possible at all?

Reply:
As far as I can tell, this script is not running anywhere so I can't verify what the API returns. Maybe it'd be better to just delete it?

Reply:
> Maybe it'd be better to just delete it?

Yes...